upgrade_test.py

```diff
@@ -765,11 +768,15 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
 
         # Verify sstabledump
         InfoEvent(message='Starting sstabledump to verify correctness of sstables').publish()
+        if ComparableScyllaVersion(self.db_cluster.nodes[0].scylla_version) < "2023.2" or \
+                ComparableScyllaVersion(self.db_cluster.nodes[0].scylla_version) < "5.4":
```

Comment:
This condition will always provide `True` for any OSS version.
It is bug.
Need to reorder condition parts or add a check for enterprise-type in addition to existing conditions.

Reply:
good catch, as some point we might want to add something to ComparableScyllaVersion, to help doing those in less verbose manner